package.json

```diff
@@ -10,23 +10,28 @@
     "scripts": {
         "prettier-format": "prettier --config .prettierrc './**/*.ts' --write",
         "prettier-watch": "onchange './**/*.ts' -- prettier --write {{changed}}",
-        "lint": "eslint . --ext .ts",
+        "lint": "eslint . --ext .ts,.tsx",
+        "lint:fix": "eslint . --ext .ts,.tsx --fix",
         "install:all": "npm i && cd packages/webapp && npm i && cd ../..",
-        "ts-build": "./node_modules/typescript/bin/tsc -b --clean packages/server packages/cli && ./node_modules/typescript/bin/tsc -b tsconfig.build.json && ./packages/webapp/node_modules/typescript/bin/tsc -b packages/webapp/tsconfig.json",
+        "ts-build": "tsc -b --clean packages/server packages/cli && tsc -b tsconfig.build.json && tsc -b packages/webapp/tsconfig.json",
```

Comment:
That is going to use a global install of tsc, right? Is it better than using the repo one?

Reply:
Within the context of a npm script the executables in the node_modules/.bin directory are added to the `PATH` for execution so the repo tsc will be used here.

Reference: https://docs.npmjs.com/cli/v7/using-npm/scripts#path